docs/platforms/javascript/guides/nuxt/install/top-level-import.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Top-level import
+sidebar_order: 2
+description: "Learn about how the Nuxt SDK adds a top-level import to the build output."
+---
+
+## Understanding top-level `import`
+
+Sentry needs to be initialized before the application starts.
+If the default way of adding an <PlatformLink to="/install/cli-import">`--import` CLI flag</PlatformLink> flag does not work for you,
+set up the SDK for adding a top-level `import`. This will import the Sentry server-side config at the top of the server entry file.
```

Comment:
l: maybe being a bit more specific here is less confusing?

```suggestion
set up the SDK for adding a top-level `import`. This will import the Sentry server-side config at the top of the Nuxt server entry file.
```